### ISSUE TITLE
OVL tool does not correctly detect Python 3.8

### ISSUE THREAD

**Issue by toxicantidote** [OpenNaja/cobra-tools#44]
The **check_version** function in _ovl_tool_gui.py_ does not correctly detect Python 3.8, and reports "Python 3.7.6+ x64 bit is expected".

This is a quick one-line fix in the function as follows:
```
	@staticmethod
	def check_version():
		is_64bits = sys.maxsize > 2 ** 32
		if not is_64bits:
			widgets.showdialog("Either your operating system or your python installation is not 64 bits.\n"
							   "Large OVLs will crash unexpectedly!")
		if sys.version_info[0] != 3 or sys.version_info[1] < 7 or (sys.version_info[1] == 7 and sys.version_info[2] < 6):
			widgets.showdialog("Python 3.7.6+ x64 bit is expected!")
```

Note that this won't accept Python 4.x if it ever happens,  but that seems like a safe option.

**Comment by toxicantidote** [comment]
Created pull request #45

**Comment by HENDRIX-ZT2** [comment]
Fixed & merged with PR #45
